### PR TITLE
Swap timeout

### PR DESCRIPTION
**Pull request by lawlawlaw**
* Timeout if swap does not progress
* Update to the batch transfer commitment to make it compatible with changes to the mercury server

### CODE REVIEW

File: src/settings.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "min_anon_set": 3,
-  "state_entity_endpoint": "https://beta.mercurywallet.io",
-  "swap_conductor_endpoint": "https://beta.mercurywallet.io",
+  "state_entity_endpoint": "http://0.0.0.0:8000",
```
Comment (86667) [inline src/settings.json:3]: change

File: src/wallet/swap/swap.ts [hunk 2]
```diff
@@ -115,13 +125,16 @@ export const doSwap = async (
     await delay(3)
   }
 
-  let bss = await get_blinded_spend_signature(http_client, swap_id.id, statecoin.statechain_id);
-  typeforce(types.BlindedSpendSignature, bss);
```
Comment (86667) [inline src/wallet/swap/swap.ts:119]: did you mean to remove the typeforce check?

File: src/wallet/swap/swap.ts [hunk 3]
```diff
@@ -178,6 +193,27 @@ export const doSwap = async (
   return statecoin_out;
 }
 
+export const make_swap_commitment = (statecoin: any, 
+  swap_info: any, wasm_client: any): BatchData => {
+
+  let commitment_str: string = statecoin.statechain_id;
+  console.log("swaps - swap_info: ", swap_info);
```
Comment (86667) [inline src/wallet/swap/swap.ts:200]: logs. remove or format nicely and use log.info/debug

**Review by 86667 — CHANGES_REQUESTED** [review]

File: src/settings.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+  "swap_conductor_endpoint": "http://0.0.0.0:8000",
```
Comment (tomt1664) [inline src/settings.json:4]: Revert to beta.mercurywallet.io for release

**Review by tomt1664 — APPROVED** [review]

File: src/wallet/swap/swap.ts (hunk 2 above)
Comment (lawlawlaw) [reply src/wallet/swap/swap.ts:119]: yes - typeforce is done in the function

File: src/wallet/swap/swap.ts (hunk 3 above)
Comment (lawlawlaw) [reply src/wallet/swap/swap.ts:200]: ok

File: src/settings.json (hunk 4 above)
Comment (lawlawlaw) [reply src/settings.json:4]: ok

File: src/settings.json (hunk 1 above)
Comment (lawlawlaw) [reply src/settings.json]: ok

**Pull request merged**